### PR TITLE
Vulture(dead code) Github Action.

### LINKED ISSUE #22: Fix the Vulture (dead code) GitHub Action
Fix the following file `https://github.com/Karim-53/Compare-xAI/blob/main/.github/workflows/vulture.yml`

you can learn more about the bug, for example here: `https://github.com/Karim-53/Compare-xAI/actions/runs/3860871755/jobs/6581483167`

this link could help `https://github.com/anaynayak/python-vulture-action`

### PR DESCRIPTION
**Pull request by Rishav1707**
## This PR closes the following issue:
closes #22 

## Changes made:
- Added the `vulture args` in the `vulture.yml` file.

### CODE REVIEW

**Comment by Rishav1707** [comment]
@Karim-53 Please review my changes...

**Comment by Karim-53** [comment]
Hi @Rishav1707 

Thanks for your help ^^

I think the bug is not fixed as you can check in your fork: https://github.com/Rishav1707/Compare-xAI/actions/runs/3870782041/jobs/6598003269
image

File: .github/workflows/vulture.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-name: Vulture (dead code)
 on: [push]
```
Comment (Karim-53) [inline .github/workflows/vulture.yml:1]:
```suggestion
name: Vulture (dead code)
on: [push]
```

File: .github/workflows/vulture.yml [hunk 2]
```diff
@@ -12,4 +11,4 @@ jobs:
         uses: anaynayak/python-vulture-action@v1.0
         id: vulture
         with:
-          vulture-args: --min-confidence 90
+          vulture-args: example --min-confidence 90
```
Comment (Karim-53) [inline .github/workflows/vulture.yml:14]:
```suggestion
          vulture-args: ./ --min-confidence 90
```

**Review by Karim-53 — COMMENTED** [review]
accept these two

**Comment by Rishav1707** [comment]
Why this PR is not showing merged in my profile?

**Pull request merged**